### PR TITLE
Update the groups to the default god flags

### PR DESCRIPTION
**Pull request by danyelvarejao1**

### CODE REVIEW

**Comment by pegason** [comment]
I dont see the need for such a change.

**Comment by danielsalim1** [comment]
It is obvious that it is necessary to change, gods can not be limitations, even more so for an attack on a monster.

**Comment by marcinho1994** [comment]
LGTM

**Comment by pegason** [comment]
I find it more interesting to set the value to 0 than to remove. That's what I meant.

**Comment by danielsalim1** [comment]
Oh yes, you're right. I did not pay attention to that.

**Comment by DSpeichert** [comment]
I'd suggest also adding senior tutor (ID: 2), gamemaster (ID: 3) and Community Manager (ID: 4) positions with corresponding privileges - gamemaster won't be able to engage in fights or pick up items. #2248

File: data/XML/groups.xml [hunk 1]
```diff
@@ -1,7 +1,132 @@
...
+			<flag canillusionall="0" />
+			<flag cansenseinvisibility="0" />
+			<flag ignoredbymonsters="0" />
+			<flag notgaininfight="0" />
+			<flag hasinfinitemana="0" />
+			<flag hasinfinitesoul="0" />
+			<flag hasnoexhaustion="0" />
+			<flag cannotusespells="0" />
+			<flag cannotpickupitem="0" />
+			<flag canalwayslogin="0" />
```
Comment (DSpeichert) [inline data/XML/groups.xml]: `canalwayslogin` could be 1.

File: data/XML/groups.xml [hunk 2]
```diff
@@ -16,7 +141,54 @@
 			<flag hasinfinitemana="1" />
 			<flag hasinfinitesoul="1" />
 			<flag hasnoexhaustion="1" />
+			<flag cannotusespells="1" />
+			<flag cannotpickupitem="1" />
 			<flag canalwayslogin="1" />
+			<flag canbroadcast="1" />
+			<flag canedithouses="1" />
+			<flag cannotbebanned="1" />
```
Comment (DSpeichert) [inline data/XML/groups.xml]: `cannotbebanned` should be 0

File: data/XML/groups.xml [hunk 3]
```diff
@@ -16,7 +141,54 @@
...
+			<flag notgaininfight="1" />
+			<flag hasinfinitemana="1" />
+			<flag hasinfinitesoul="1" />
+			<flag hasnoexhaustion="1" />
+			<flag cannotusespells="0" />
+			<flag cannotpickupitem="0" />
+			<flag canalwayslogin="1" />
+			<flag canbroadcast="1" />
+			<flag canedithouses="1" />
+			<flag cannotbebanned="1" />
```
Comment (DSpeichert) [inline data/XML/groups.xml]: `cannotbebanned` should be 0

File: data/XML/groups.xml (hunk 2 above)
Comment (ranisalt) [reply data/XML/groups.xml]: There was once a gamemaster called Deathwalker that was banished while still a gamemaster. Just for the curious :stuck_out_tongue:

File: data/XML/groups.xml (hunk 1 above)
Comment (ranisalt) [reply data/XML/groups.xml]: That means ability to skip wait list or login with server closed?

**Comment by Zbizu** [comment]
outdated/conflicting with current groups.xml

**Comment by DSpeichert** [comment]
This look abandoned.

**Pull request closed without merging**